### PR TITLE
Update 2019-02-28.md

### PR DESCRIPTION
**Pull request by ImarinArwen**

### CODE REVIEW

**Comment by Teifun2** [comment]
I dont think you need to open a pull request.

**Review by Teifun2 — APPROVED** [review]

**Comment by marcelluethi** [comment]
@ImarinArwen Thanks for providing these notes

The way we work together on Github is not yet clearly defined. We will have to figure out as a group how we organize this. We have planned a discussion on this topic in week 4. 

For the moment just work the way you find is appropriate. If you think you need approval or a second pair of eyes to look over it, you can do a PR. If you want to commit directly, it is also fine.

**Pull request merged**
